lib/components/data/plots/CWMSPlot.jsx

```diff
@@ -12,77 +12,128 @@ const config_v2 = new Configuration({
 });
 const ts_api = new TimeSeriesApi(config_v2);
 
+
+const config_level = new Configuration({
+  headers: {
+    accept: "*/*",
+  },
+});
+
+const level_api = new LevelsApi(config_level)
+
+
+
 export default function CWMSPlot({
   tsids,
```

Comment:
Feels redundant to have this in addition to a `tsid` in `timeseriesParams`.  Maybe drop this and just reference tsids from the other object?